src/Core/Domain/Cart/Query/GetCartForOrderCreation.php

```diff
@@ -56,4 +61,34 @@ public function getCartId(): CartId
     {
         return $this->cartId;
     }
+
+    /**
+     * @return bool
+     */
+    public function hideDiscounts(): bool
+    {
+        return $this->hideDiscounts;
+    }
+
+    /**
+     * When hideDiscounts is set to TRUE,
+     * Gifted products are in a separate line from other products which are charged for
+     * The price of any gifted products is not included in the overall discounts, total products and cart total
+     * Shipping is set to 0 if there is a free_shipping cart rule
+     *
+     * Otherwise,
+     * There is one line per product type and the gifted quantity will be included to the product quantity in the cart.
+     * Shipping has its original price, and if it's free, the shipping value will be added as a discount
+     * Any gifted products' price will be included as a discount
```

Comment:
Sorry 😁 
```suggestion
     * When hideDiscounts is set to TRUE,
     * Gift products are in a separate line from other products which are charged for
     * The price of any gift products is not included in the overall discounts, total products and cart total
     * Shipping is set to 0 if there is a free_shipping cart rule
     *
     * Otherwise,
     * There is one line per product type, any gift products will be included in the quantity of charged products, but the price of gift products will appear as a discount
     * Shipping has its original price, and if it's free, the shipping value will be added as a discount
```